### PR TITLE
default organisation setting

### PR DESCRIPTION
**Pull request by tejaswinichile**
Issue #532 
1. An organization has one setting record to save base_url and visibility of organization setting page
2. Create a default setting record when an organization gets created.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -120,6 +120,14 @@
     t.index ["slug"], name: "index_organizations_on_slug", unique: true
   end
 
+  create_table "settings", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.string "base_url"
+    t.boolean "visibility", default: false, null: false
+    t.uuid "organization_id"
```
Comment (neerajdotname) [inline db/schema.rb]: @tejaswinichile Please make it not nullable and please add a foreign key for it.

Comment (tejaswinichile) [reply db/schema.rb]: Foreign_key is present for the same >> https://github.com/bigbinary/acehelp/pull/540/files#diff-1acd2e7e27a227829d5d14a91c863bb6R202. I will make it `null: false`.

**Comment by tejaswinichile** [comment]
@prathamesh-sonpatki _a Pease check.

File: app/graphql/mutations/organization_mutations.rb [hunk 2]
```diff
@@ -19,6 +19,7 @@ class Mutations::OrganizationMutations
 
         if new_org
           organization = new_org
+          Setting.where(organization_id: organization.id).first_or_create!
```
Comment (prathamesh-sonpatki) [inline app/graphql/mutations/organization_mutations.rb]: I think `after_create` callback would be better here to create related setting record for an organization.

**Comment by prathamesh-sonpatki** [comment]
@tejaswinichile _A

**Comment by tejaswinichile** [comment]
@prathamesh-sonpatki _a

File: test/graphql/mutations/organization_mutations_test.rb [hunk 3]
```diff
@@ -64,4 +64,15 @@ class Mutations::OrganizationMutationsTest < ActiveSupport::TestCase
       })
     end
   end
+
+  test "settings present after organization create" do
```
Comment (prathamesh-sonpatki) [inline test/graphql/mutations/organization_mutations_test.rb]: Please rename it to `Default settings are created after an organization is created`. We need to have proper English statements in the test descriptions.

File: test/graphql/mutations/organization_mutations_test.rb [hunk 4]
```diff
@@ -64,4 +64,15 @@ class Mutations::OrganizationMutationsTest < ActiveSupport::TestCase
...
+    result = AceHelp::Client.execute(@common_org_query, input: {
+      user_id: @user.id,
+      name: "New Organisation Name",
+      email: "org_general_email@gmail.com"
+    })
+    organization_id = result.data.add_organization.organization.id
+    organization = Organization.find_by!(id: organization_id)
+    assert organization.setting
```
Comment (prathamesh-sonpatki) [inline test/graphql/mutations/organization_mutations_test.rb]: Please add new line before this.

Basically, we should write tests like this:

```
setup

assertions
```

It becomes much more readable that way.

**Comment by prathamesh-sonpatki** [comment]
@tejaswinichile _A

**Comment by tejaswinichile** [comment]
@prathamesh-sonpatki _a Please check.

**Pull request merged**